prov/util/src/util_buf.c

```diff
@@ -47,32 +47,34 @@ enum {
 
 int ofi_bufpool_grow(struct ofi_bufpool *pool)
 {
-	void *buf;
-	int ret;
-	size_t i;
 	struct ofi_bufpool_region *buf_region;
 	ssize_t hp_size;
 	struct ofi_bufpool_ftr *buf_ftr;
+	void *buf;
+	int ret;
```

Comment:
should 'ret' be initialized to 0?

Reply:
We're okay here.  It'll be set if we take an error path, otherwise we'll hit a return 0.  Leaving this _unset_ actually helps us, because if we leave it unset and get a warning, that means we're taking an error path without having set ret to a proper error code.

Reply:
Blame the snow :) somehow I missed the normal return path.